pkg/operator/options/options.go

```diff
@@ -0,0 +1,56 @@
+package options
+
+import (
+	"context"
+	"errors"
+	"flag"
+	"fmt"
+	"os"
+
+	coreoptions "sigs.k8s.io/karpenter/pkg/operator/options"
+)
+
+func init() {
+	coreoptions.Injectables = append(coreoptions.Injectables, &Options{})
+}
+
+type optionsKey struct{}
+
+type Options struct {
+	ClusterAPIKubeConfigFile string
+}
+
+func (o *Options) AddFlags(fs *coreoptions.FlagSet) {
+	fs.StringVar(&o.ClusterAPIKubeConfigFile, "cluster-api-kubeconfig", "", "The path to the cluster api manager cluster kubeconfig file.  Empty string for workload cluster config.")
```

Comment:
one minor nit, i think we should be more explicit about the default behavior.

```suggestion
	fs.StringVar(&o.ClusterAPIKubeConfigFile, "cluster-api-kubeconfig", "", "The path to the cluster api manager cluster kubeconfig file.  Defaults to service account credentials if not specified.")
```